src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java

```diff
@@ -85,12 +84,12 @@ public void copiesFromCharArrayToOutput() throws IOException {
         final String input =
             "Hello, товарищ output #1 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
+        new LengthOf(
+            new TeeInput(input.toCharArray(), new OutputTo(output), StandardCharsets.UTF_8.name())
```

Comment:
@a-korzhov you mis-copied it, you added a charset parameter while it wasn't here before